### ISSUE TITLE
Unpeeled Git tag references cause NPE when listing tags

### ISSUE THREAD

**Issue by malesch** [boot-clj/boot#399]
`boot.git/ls-tags` throws a NullPointerException if an unpeeled tag reference exists.

Example:

```
> mkdir prj
> cd prj
> git init
> touch foo.txt
> git add . ; git commit -m 'init'
> git tag foo1

> boot repl
boot.user=> (boot.git/ls-tags)

java.lang.NullPointerException:
boot.user=>
```

**Comment by malesch** [comment]
Fix proposal in PR #400

**Comment by micha** [comment]
Fixed in f2cbec5c5deeddc9523f29b1b95ae1b1e9e24a22
